browser.go

```diff
@@ -119,6 +119,18 @@ func NewBrowser(ctx context.Context, urlstr string, opts ...BrowserOption) (*Bro
 	return b, nil
 }
 
+// Process returns the process object of the browser.
+// It could be nil when the browser is allocated with RemoteAllocator.
+// It can be used to control the memory consumed by the browser process.
```

Comment:
I meant an example to show how to control the memory consumed by the browser process. It's not necessary to be put into the source code, it's okay to describe it in the pull request message.

Please note that #566 is abandoned because there is not a concrete purpose. We need an example to show that why we need to export this object.

Reply:
Just for monitoring. We use this to export prometheus metrics with the consumed memory of sessions, for displaying on the Grafana dashboard.

Reply:
It makes sense now. Thank you very much! :+1:

Reply:
Let's change the line to:

```go
// It could be useful for a monitoring system to collect process metrics of the browser.
// (see https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#NewProcessCollector for an example).
```

What do you think?

Reply:
Let's change this line to:

```go
// It could be useful for a monitoring system to collect process metrics of the browser process.
// (see https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#NewProcessCollector for an example)
```

What do you think?

Reply:
Changed